parsers/ShortcodeParser.php

```diff
@@ -612,4 +620,10 @@ function ($matches) use ($tags, $parser) {
 
 		return $content;
 	}
+
+	public static function getShortcodes($instance = 'default')
```

Comment:
Do we need this method? If not, I would remove it. I think we want to keep the number of static accessors to a minimum as they make it easy to increase coupling.